src/UserForm.js

```diff
@@ -55,6 +58,7 @@ class UserForm extends React.Component {
         component={Select}
         fullWidth
         onChange={onChangeAddress}
+        validate={this.requireDeliveryAddress}
```

Comment:
if a selection is required, shouldn't we pass `required` to its `<Field>` as well?

Reply:
Good catch. Will add it.